lib/smart_city/dataset/technical.ex

```diff
@@ -30,6 +31,7 @@ defmodule SmartCity.Dataset.Technical do
   @derive Jason.Encoder
   defstruct allow_duplicates: true,
             authHeaders: %{},
+            authBody: %{},
```

Comment:
May want to update the moduledoc in `SmartCity.Dataset` to show this field is a part of the struct... not sure it's the right place for that doc, but that's where we've been doing it this far.

Reply:
True

Reply:
Ah, will do.